pitch.md

```diff
@@ -14,31 +23,170 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+From time to time, teachers and students of geometry and associated fields are
+interested in creating constructions. While sometimes they understand what they
+are creating, often times they are creating the idea as they make the drawing.
+
+It is also worth noting that while frequently users want these constructions
+rendered into drawings, the objects they are describing are geometric in
+nature - not inherently graphical. This means that sometimes geometrically
+distinct ideas may be spatially close. If input is spatial then this can result
+in geometric imprecision. For example, consider drawing a line from a triangle
+vertex to the midpoint of the opposing side vs bisecting that angle. Often
+these two actions are spatially similar (the lines end up close to each other).
+
+Current solutions to the problem fail to be both easy to use and geometrically
+precise. `tkz-euclid` is geometrically precise but very cryptic, and there are
+various visual construction engines which are easy to use but rely on spatial
+input and are thus geometrically imprecise in some cases.
+
+Furthermore, none of the above systems allow the user to see what steps they
+might take next.
```

Comment:
I have not, but I imagine I might need to at some point.

Perhaps machine learning could be helpful here? Or a built-in rule